packages/next/src/builders/dev-server/dev-server.impl.ts

```diff
@@ -6,16 +6,12 @@ import {
   scheduleTargetAndForget
 } from '@angular-devkit/architect';
 import { JsonObject } from '@angular-devkit/core';
-import {
-  PHASE_DEVELOPMENT_SERVER,
-  PHASE_PRODUCTION_SERVER
-} from 'next/dist/next-server/lib/constants';
-import startServer from 'next/dist/server/lib/start-server';
-import NextServer from 'next/dist/server/next-dev-server';
+import * as http from 'http';
+import next from 'next';
 import * as path from 'path';
 import { from, Observable, of } from 'rxjs';
 import { switchMap, concatMap, tap } from 'rxjs/operators';
-import { prepareConfig } from '../../utils/config';
+import { StartServerFn } from '../../..';
```

Comment:
As with the other PR based on this code, I might suggest switching around where some of the types live, so that `index.ts` is more of a barrel file - this import could then just be `import { StartServerFn } from './start-server'` (or similar, I'm not making any concrete declarations about where it should live)

Reply:
Done